### ISSUE TITLE
Payload Functions: Encoder test fails while function is correct

### ISSUE THREAD

**Issue by romeovs** [TheThingsNetwork/ttn#239]
If I use this for Encoder:

```js
function Encoder(obj) {
	return "Hi".split('').map(function(char) {
		return char.charCodeAt();
	});
}
```

Test fails with:

```
Encoder not valid: should return an Array of integer numbers
```

But run this in the browser or Node.js and you do get an Array of integer numbers:
```js
[ 72, 105 ]
```

**Comment by romeovs** [comment]
closing as there is a PR in upstream

**Comment by romeovs** [comment]
closed by #240
